### PR TITLE
Admin panel

### PR DESCRIPTION
**Pull request by eun-chae-s**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -15,10 +30,32 @@ function App() {
...
           <Route path="bookDetails" element={<BookDetailsPage />}></Route>
-          <Route path="exerciseDetails" element={<ExerciseDetailsPage />}></Route>
+          <Route
+            path="exerciseDetails"
+            element={<ExerciseDetailsPage />}
+          ></Route>
           <Route path="editExercise" element={<EditExercisePage />}></Route>
         </Routes>
       </SearchProvider>
+      <Routes>
```
Comment (junseomoon9) [inline src/App.js:40]: Why do these routes have to be separated from the existing routes that is already in place?

File: src/api/requests.js [hunk 2]
```diff
@@ -1,6 +1,82 @@
...
+    .catch((err) => {
+      console.log(err);
+      setSuccess(false);
+    });
+}
+
+/** get all the requests */
+export function getAllRequests(setRequests) {
+  axios.get(server_url + "api/requested/").then((res) => {
+    console.log(res.data.results);
```
Comment (junseomoon9) [inline src/api/requests.js:27]: Remove console.logs

Comment (eun-chae-s) [reply src/api/requests.js:27]: already did in the recent comment!

Comment (junseomoon9) [reply src/api/requests.js:27]: Do you mean you commited the changes locally but haven't pushed it yet?

Comment (eun-chae-s) [reply src/api/requests.js:27]: I've already pushed it, and it appears here:)
Screen Shot 2022-12-10 at 1 51 05 PM

Comment (rebplane) [reply src/api/requests.js:27]: @eun-chae-s looks like you removed the console.log for getAllTags but some still exist in requests.js

Comment (eun-chae-s) [reply src/api/requests.js:27]: oohh let me check again

**Review by junseomoon9 — APPROVED** [review]

**Pull request merged**